client.go

```diff
@@ -22,6 +23,10 @@ type Client struct {
 	debugLogFunc LogFunc
 }
 
+func LintError() {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function LintError should have comment or be unexported